config/config.go

```diff
@@ -705,6 +707,15 @@ type StorageConfig struct {
 	// If true, we'll first delete all tables in the DB to
 	// force a full re-index of the blockchain.
 	WipeStorage bool `koanf:"DANGER__WIPE_STORAGE_ON_STARTUP"`
+
+	// Postgres is the postgres specific configuration to use.
+	Postgres *PostgresConfig `koanf:"postgres"`
+}
+
+// PostgresConfig is the postgres specific configuration to use when using the postgres backend.
+type PostgresConfig struct {
+	// MaxConnections is the maximum number of connections in the database connection pool.
+	MaxConnections *int32 `koanf:"max_connections"`
```

Comment:
To my understanding, this parameter can be configured via a connection string (specified in either key/value or URI format), such as:
`user=jack password=secret host=pg.example.com port=5432 dbname=mydb sslmode=verify-ca pool_max_conns=10`

Therefore, there's no need to expose it specifically via the Nexus' config.

See here for details:
- https://www.postgresql.org/docs/current/libpq-connect.html#LIBPQ-CONNSTRING
- https://github.com/jackc/pgx/blob/v4.18.3/pgxpool/pool.go#L305

Reply:
Hm you're right. Will remove it in a future release, to avoid duplication of possible configuration.